### PR TITLE
Collection of fixes to quote text and author names

### PR DESCRIPTION
**Pull request by dannya**

### CODE REVIEW

**Review by nelsonic — APPROVED** [review]
@dannya thank you for the great fixes and additions. 🙏 🎉 

When you feel this is ready for merge, LMK. 👌

**Comment by dannya** [comment]
@nelsonic thanks - I can't merge it as I don't have write access to the repository.

**Comment by nelsonic** [comment]
@dannya in general we _never_ merge our own PRs. 😉 
Was just asking if you thought you were adding/fixing anything `else` ... 🧑‍💻 

Merging this one. :shipit: 
Will publish new packages when I'm back at my desk. ⏳

**Pull request merged**
